src/demo-app/chips/chips-demo.html

```diff
@@ -46,22 +46,20 @@
       </p>
 
 
-      <md-form-field>
-        <md-chip-list mdPrefix #chipList1>
+      <md-chip-list #chipList1>
```

Comment:
This seems weird, putting the `md-form-field` inside the `md-chip-list`, it feels like it should be the other way around... I think it should be possible now to have the `md-chip-list` implement the `MdFormFieldControl` interface and work with a parent `md-form-field` via that interface. Is this something you've considered?